feature-libs/order/core/facade/order-history.service.ts

```diff
@@ -208,4 +208,11 @@ export class OrderHistoryService implements OrderHistoryFacade {
   resetCancelOrderProcessState(): void {
     return this.store.dispatch(new OrderActions.ResetCancelOrderProcess());
   }
+
+  /**
+   * Returns the order details loading flag
+   */
+  getOrderDetailsLoading(): Observable<boolean> {
+    return this.store.pipe(select(OrderSelectors.getOrderDetailsLoading));
```

Comment:
I'm thinking about handling the breaking changes in the most convenient way. If we add the feature flag here as returning an Observable with value `true` when we have not updated our feature level, that should handle all cases we use `isOrderLoading$`?

Reply:
I think it should return and Observable with `false`. Then it's not breaking anything below 5.2